examples/tf-azurerm-computegroup/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "resource_group_name" {
+    description = "The name of the resource group in which the resources will be created"
+    default = "vmssrg"
+}
+
+variable "location" { 
+    description = "The location where the resources will be created"
+    default = "West US"
+}
+
+variable "vm_size" {
+    default = "Standard_A0"
+    description = "Size of the Virtual Machine based on Azure sizing"
+}
+
+variable "admin_username" {
+    description = "The admin username of the VMSS being deployed"
+    default = "azureuser"
+ }
+
+variable "admin_password" {
+    default = ""
+}
+
+variable "ssh_key" {
+    description = "Path to the public key to be used for ssh access to the VM"
+    default = "~/.ssh/demo_key.pub"
```

Comment:
might be worth defaulting this to `~/.ssh/id_rsa.pub` given that's the default ssh key path?